arlo-client/src/components/CreateAudit.tsx

```diff
@@ -22,17 +22,30 @@ const Wrapper = styled.div`
   }
 `
 
+const AuditLink = styled(Link)`
+  display: block;
+  margin: 10px;
+
+  &:first-of-type {
+    margin-top: 30px;
+  }
+`
+
 const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
-  const { isAuthenticated } = useAuthDataContext()
+  const { isAuthenticated, meta } = useAuthDataContext()
 
   const [loading, setLoading] = useState(false)
   const onClick = async () => {
     try {
       setLoading(true)
+      const data = isAuthenticated
+        ? { organization: meta!.organizations[0].id }
```

Comment:
@benadida Here you go! :slightly_smiling_face: Let me know if there's anything else you need on this endpoint.